core/src/VeyonConfigurationProperties.h

```diff
@@ -119,7 +119,8 @@
 	OP( VeyonConfiguration, VeyonCore::config(), bool, enforceSelectedModeForClients, setEnforceSelectedModeForClients, "EnforceSelectedModeForClients", "Master", false, Configuration::Property::Flag::Standard )	\
 	OP( VeyonConfiguration, VeyonCore::config(), bool, autoOpenComputerSelectPanel, setAutoOpenComputerSelectPanel, "AutoOpenComputerSelectPanel", "Master", false, Configuration::Property::Flag::Standard )	\
 	OP( VeyonConfiguration, VeyonCore::config(), bool, confirmUnsafeActions, setConfirmUnsafeActions, "ConfirmUnsafeActions", "Master", false, Configuration::Property::Flag::Standard )	\
-
+	OP( VeyonConfiguration, VeyonCore::config(), bool, ShowFeatureWindowsOnSameScreen, setShowFeatureWindowsOnSameScreen, "ShowFeatureWindowsOnSameScreen", "Master", false, Configuration::Property::Flag::Standard )	\
```

Comment:
Too much changed here ;-) the actual config property getters should start lower case, only the property name had to be changed. See the other properties as an example.

Reply:
Sigh... I need to be more careful. Thank you for your patience.

Reply:
Thank you for your patience ;-)